sci-mathematics/minisat/minisat-2.2.0-r4.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Small yet efficient SAT solver with reference paper"
+HOMEPAGE="http://minisat.se/Main.html"
+SRC_URI="http://minisat.se/downloads/${P}.tar.gz
+	doc? ( http://minisat.se/downloads/MiniSat.pdf )"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+LICENSE="MIT"
+
+IUSE="debug doc extended-solver"
+
+DEPEND="sys-libs/zlib"
+RDEPEND="${DEPEND}"
+
+DOCS=( README doc/ReleaseNotes-2.2.0.txt )
+
+S=${WORKDIR}/${PN}
+
+src_prepare() {
+	default
+	sed -e "s/\$(CXX) \$^/\$(CXX) \$(LDFLAGS) \$^/" \
+		-i -e "s|-O3|${CFLAGS}|" mtl/template.mk || die
+}
+
+src_configure() {
+	myconf=$(usex debug "d" "r")
+	myext=$(usex debug "debug" "release")
+	mydir=$(usex extended-solver "simp" "core")
```

Comment:
there's no potential for accidental word splits, hence remove all the quotation marks in the `usex`'s